cves/kernel/CVE-2015-2041.yml

```diff
@@ -56,6 +56,8 @@ description_instructions: |
   Your target audience is people just like you before you took any course in
   security
 description:
+  There was a sysctl table (net/llc/sysctl_net_llc.c) that used a wrong datatype for one of the parameters,
+  allowing users to obtain information from kernel memory by accessing a sysctl entry.
```

Comment:
May be worth mentioning here that that data is accessible and can even be edited with the same permissions needed to do the exploit.